internal/configconverter/discovery.go

```diff
@@ -158,6 +161,25 @@ func getMetricsPipelineAndReceivers(service map[string]any) (map[string]any, []a
 	return metricsPipeline, metricsReceivers, nil
 }
 
+func setEntitiesPipelineReceivers(pipelines map[string]any, discoReceivers []any) {
+	ep, ok := pipelines["logs/entities"].(map[string]any)
+	if !ok {
+		// Entities pipeline not set, nothing to do.
```

Comment:
would a log statement make sense here?

Reply:
Yeah makes sense. But we currently don't pass a logger here. I'll address that separately